textdb/textdb-dataflow/pom.xml

```diff
@@ -21,6 +21,11 @@
             <artifactId>junit</artifactId>
             <version>${junit.version}</version>
         </dependency>
+         <dependency>
+			<groupId>com.google.re2j</groupId>
+			<artifactId>re2j</artifactId>
+			<version>1.1</version>
+		</dependency>
```

Comment:
@zuozhi : is this really an issue or because github doesn't show the format properly? No matter, please add a reply.